asyncnsq/http/base.py

```diff
@@ -34,6 +34,9 @@ async def perform_request(self, method, url, params, body):
                                                data=_body)
         except Exception as tmp:
             print('exception', tmp)
+            # FIXME how it should be handled?
+            raise
+
         resp_body = await resp.text()
```

Comment:
@aohan237 on exception `await resp.text()` would fail since `resp` is none. What is desirable behavior — response text or raise an error?

Reply:
maybe we should leave the exception as the origin by not catching it, let user to decide what to do if exception happends